src/test/java/com/wikia/webdriver/common/core/exceptions/TestFailed.java

```diff
@@ -0,0 +1,12 @@
+package com.wikia.webdriver.common.core.exceptions;
+
+/**
+ * Created by Ludwik on 2015-04-14.
+ */
+public class TestFailed extends Throwable {
```

Comment:
Question: Is there something specific of this class that it needs to extend Throwable and not Exception?